cypress/e2e/16_command_menu.cy.ts

```diff
@@ -0,0 +1,14 @@
+/// <reference types="cypress" />
+
+//** Tests app gallery feature.*/
+
+describe("Command test", () => {
+  it("Command test", () => {
+    cy.visit("/").wait(1000);
+
+    cy.get('[data-testid="react-flow"]').trigger("keydown", {
```

Comment:
Maybe add a check after this to make sure that something actually happened?

Reply:
I was planning to check it visually using Percy. Do you think it's a bad idea?

Reply:
Yeah that's good